css/styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+body#home {
+  margin: 0;
+  background-color: #ADD8E6;
+  color: white;
+}
+
+body#about {
+  margin: 0;
+  background-color: white;
+  color: gray;
+}
+
+body#experience {
+  margin: 0;
+  background-color: white;
+  color: gray;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+li {
+  display: inline;
+  float: right;
+}
+
+
+a.general {
+  text-decoration: none;
+  color: #ADD8E6;
+}
+
+a.navhome {
+  color: white;
+  padding: 25px 50px;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+}
+
+a.navhome:link {
+  color: #000080;
+  padding: 25px 50px;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+
+}
+
+a.navhome:hover {
+  color: #0000A0;
+  padding: 25px 50px;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+
+}
+
+a.nav {
+  color: #ADD8E6;
+  padding: 25px 50px;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+}
+
+a.nav:link {
+  color: #000080;
+  padding: 25px 50px;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+
+}
+
+a.nav:hover {
+  color: #0000A0;
+  padding: 25px 50px;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+
+}
+
+h4 {
+  text-align: center;
+  font-size: 75px; 
+  text-transform: uppercase;
+  font-family: 'Bungee Hairline', cursive;
+  margin-top: 250px;
+  margin-bottom:15px;
+}
+
+h5.title {
+  line-height: -5;
+  text-align: center;
+  font-size: 29.5px; 
+  margin-top: 0px;
+  margin-bottom: 0px;
+  text-transform: uppercase;
+
+}
+
+h1 {
+  text-transform: uppercase;
+}
+
+p {
+  font-size: 20px;
+}
+
+img.header {
+    float: left;
+    padding-right: 20px;
+    padding-bottom: 40px;
+}
+
+img.inline {
+  float: left;
+  padding-right: 20px;
+}
+
+div {
+  text-align: left;
+}
+
+.main > p {
+  font-weight: bold;
+}
+
+.main {
+  margin-top: 20;
+}
+
+.section {
+  margin-top: -15;
+}
+
+.experience {
```

Comment:
@jiniekwak since the are the same rules you could have combined the selectors `.experience, .about { ...rules...}`